cloudant-client/src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -265,8 +270,9 @@ public URI getBaseUri() {
      * Get the reported server version from the welcome message metadata.
      *
      * @return Cloudant server version.
-     * @see <a target="_blank" href="https://docs.cloudant.com/advanced.html#get-/">Welcome
-     * message</a>
+     * @see <a target="_blank"
+     * href="https://console.bluemix.net/docs/services/Cloudant/api/advanced.html#-get-">

```

Comment:
Similarly to the above "advanced" page link this anchor doesn't seem to work directly, but in this case it is a bit less of a problem since the root `GET` is the first element on the page anyway.